release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2775,6 +2775,53 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.42 --pullspecs
+----
+
+[id="ocp-4-15-42-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when the webhook token authenticator was enabled and had the authorization type set to `None`, the {product-title} web console would consistently crash. With this release, a bug fix ensures that this configuration does not cause the {product-title} web console to crash. (link:https://issues.redhat.com/browse/OCPBUGS-46482[*OCPBUGS-46482*])
+
+* Previously, when you attempted to use {olm-first} to upgrade an Operator, the upgrade was blocked and an `error validating existing CRs against new CRD's schema` message was generated. An issue existed with {olm}, whereby {olm} erroneously identified incompatibility issues validating existing custom resources (CRs) against the new Operator version's custom resource definitions (CRDs). With this release, the validation is corrected so that Operator upgrades are no longer blocked. (link:https://issues.redhat.com/browse/OCPBUGS-46479[*OCPBUGS-46479*])
+
+* Previously, the images for custom OS layering were not present when the OS was on {rhcos-short} 4.15, preventing some customers from upgrading from {rhcos-short} 4.15 to {rhcos-short} 4.16. This release adds ACR and GCR image credential provider RPMs to {rhcos-short} 4.15. (link:https://issues.redhat.com/browse/OCPBUGS-46063[*OCPBUGS-46063*])
+
+* Previously, it was impossible to configure your Amazon Web Services DHCP option set with a custom domain name containing a trailing period, as trailing periods were not allowed in a Kubernetes object name. With this release, trailing periods are allowed in a domain name in a DHCP option set. (link:https://issues.redhat.com/browse/OCPBUGS-46034[*OCPBUGS-46034*])
+
+* Previously, when `openshift-sdn` pods were deployed during the {product-title} upgrading process, the Open vSwitch (OVS) storage table was cleared. This issue occurred on {product-title} {product-version}.19 and later versions. Ports for existing pods had to be re-created and this disrupted numerous services. With this release, a fix ensures that the OVS tables do not get cleared and pods do not get disconnected during a cluster upgrade operation. (link:https://issues.redhat.com/browse/OCPBUGS-45955[*OCPBUGS-45955*])
+
+* Previously, you could not remove a finally pipeline task from the *edit Pipeline* form if you created a pipeline with only one finally task. With this release, you can remove the finally task from the *edit Pipeline* form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45950[*OCPBUGS-45950*])
```

Comment:
Suggest putting "finally" in backticks (like [here](https://tekton.dev/docs/pipelines/pipelines/)) because it looks like a typo otherwise :sweat_smile: 
```suggestion
* Previously, you could not remove a `finally` pipeline task from the *edit Pipeline* form if you created a pipeline with only one `finally` task. With this release, you can remove the `finally` task from the *edit Pipeline* form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45950[*OCPBUGS-45950*])
```

Reply:
true! great catch!